src/components/Bookmark/Bookmark.js

```diff
@@ -38,6 +38,12 @@ class Bookmark extends Component {
   }
   
```

Comment:
Trailing spaces not allowed
line 21, 31, 46, 55, 59, 64, 75, 79, 102, 107 have same error.